TestCntlrApp/src/enbApp/nb_s1ap_hndlr.c

```diff
@@ -818,6 +818,28 @@ PRIVATE S16 nbBldResetReq
    nbFillTknU8(&(ie->pres), PRSNT_NODEF);
    nbFillTknU32(&(ie->id), Sztid_ResetTyp);
    nbFillTknU32(&(ie->criticality), SztCriticalityrejectEnum);
+
+   if(resetMsgInfo->type != NB_COMPLETE_RESET && resetMsgInfo->type != NB_PARTIAL_RESET)
+   {
+      NB_LOG_ERROR(&nbCb, "Invalid Reset Type");
+      NB_FREE_EVNT(resetReqPdu);
+      RETVALUE(RFAILED);
+   }
+
+   // Allocate memory for storing UE Id pair to be reset
+   ret = cmAllocEvnt(sizeof(S1apPdu), NB_SZ_MEM_SDU_SIZE, &nbCb.mem,
```

Comment:
ret is not used anywhere. You can directly check the return type of the fun against ROK/RFAILED -> 
if ((cmAllocEvnt)!=ROK)